### ISSUE TITLE
Responsive SVG produces PNG with square width and height

### ISSUE THREAD

**Issue by ohenrik** [exupero/saveSvgAsPng#81]
It seems like the width and height property always follows the viewBox. This means that even if i have set the width and height by js and the svg is scaled. The png is left a square cropping the rest of the image.

Example:
How the svg looks:
![screen shot 2016-04-07 at 20 38 26](https://cloud.githubusercontent.com/assets/647617/14362658/cd483f2a-fd00-11e5-9f56-b60bdc225e37.png)

How the downloaded image looks:
![diagram 28](https://cloud.githubusercontent.com/assets/647617/14362677/dcb36e3a-fd00-11e5-94bd-835c931eb7a3.png)

How can i set the correct width and height attribute to get the responsive correct download?

**Comment by ohenrik** [comment]
I have found a temporary fix. Not sure if this is legit, however it works for me. 

I made two more options named `exportWidth`, and `exportHeight`. Then i used those values as width and height properies for the new export:

```
  clone.setAttribute("width", exportWidth * options.scale);
  clone.setAttribute("height", exportHeight * options.scale);
```

These are at line 167 and 168

**Comment by exupero** [comment]
You should be able to specify the `width` and `height` options. The viewbox is only used if those options are not provided.

**Comment by ohenrik** [comment]
That is correct, however if i set the normal width and heigh property i got wierd results. If i used window width and height i got only a small part of the svg. Becuase that setting is then also used to set the viewbox of the hidden svg that is used in the export. This results in incorrect responsive behaviour.
